### PR TITLE
Brotherhood Armor Sprites, Paladin Loadouts, Powerfist 'Nerf' & Other Changes

### PR DESCRIPTION
**Pull request by SouthyOsmanli**
## About The Pull Request

**Large BOS focused PR**; ~~_still working on this but publishing for feedback specifically_~~ **Should be good to go!** This is a balance change, as Paladins had little actual reason to choose anything but the assault class due to the Wattz 2k not being comparable at all to the AER-12 and the Powerfist needing to not be on **EVERY** single Paladin class. Furthermore the Powerfist was nerfed in both damage and AP, the sledge received a small buff and loadouts were changed somewhat notably by removing unsued grenades and stuff either not used, having no purpose there or simply for mild balancing .

Removed super sledge from armory due to being on a loadout and being buffed a slight bit; removed initiate armors because people are using it for power-game gear by scribes. No other faction has spare armor sitting around so, arguably, neither should BOS if it serves no purpose.

Also added sprites for the custom BOS armor that is craftable as well as made head knight armor have the correct tier of armor since it tiered weirdly (HK Scout = 6, HK regular = 7 and the HK PA = 9 for some reason? Moved HK PA to 8).

## Why It's Good For The Game

The **Brotherhood Helmet Sprites** are simply completing the custom armor the Brotherhood already gets - though the helmets do not look proper due to the sprite miss-match.

The **Paladin Loadouts** are for more variety and removing mostly unused gear from Paladins in an attempt to balance somewhat. Not every Paladin should have a powerfist due to its stats, which have been decreased, and the addition of the Big Leagues book allows for more melee-focused builds. Paladins _never_ used the Wattz 2k due to how underpowered it felt and realistically preforms compared to the AER-12 alternative. This should provide a proper trade-off. Also, Head Paladin has a new loadout to choose between Gauss, the AER-14 or a heavier version of the powerfist but with close to no AP. The reason _why_ I included Big Leauges with the powerfist is to keep its damage around the same but with less AP and make the class unique. Tested it a bit and it seemed to preform just as before mostly.

The **Super Sledge** and **Powerfist** in their own regards needed change. Powerfist's force and AP was a bit much for it, allowing concussions in one hit due to force and its high AP. This was OK at first till I realized that one can be knocked out well due to the AP damage; though it was notably better than how it was with the gas system. The sledge, on the other hand, has only gotten a VERY small buff by giving it 0.1 AP over having a total of 0. This should make it mildly more viable as it has gone unused both in BOS armory (Which, I will do a map change removing it from if the Head Paladin having it is approved) and as a overall weapon to non-BOS members.

## Changelog
:cl:
add: BOS custom sprite helmets.
add: Big Leagues 'Swatter' granter book.
removes: Initiate armor, helmets and super sledge from armory.
tweak: Powerfist stats (50->40 force, 0.8 -> 0.3 AP).
tweak: Super sledge stats (0.0 AP -> 0.1 AP).
balance: Paladin loadouts; large changes for Head Paladin, moderate ones for other Paladins.
balance: Made Head Knight PA no longer a tier 9 T-45 armor piece since Paladins only get tier 8 T-45.
balance: Mole Miner changed to not be an improved version of the powerfist since it's more common.
balance: Ballistic fist firedelay changed to try and prevent it from hitting as fast as one can click; decrease in AP slightly.
/:cl:

### CODE REVIEW

**Comment by gurking** [comment]
Everything looks good, but I do not think Paladins should get big leagues, as it's literally the best PvE perk for a faction that can already steamroll bunkers thanks to their easy access to shields and high tier armor and medicine. This will just allow them to rush bunkers and then use the high tier guns from there to round out their loadout.

Maybe give them the cut down laser tribeam, for a close-range option in combination with the power fist.

File: code/modules/clothing/head/f13head.dm [hunk 1]
```diff
@@ -405,6 +405,13 @@
 	icon_state = "t45dhelmet[on]"
 	item_state = "t45dhelmet[on]"
 
+/obj/item/clothing/head/helmet/f13/power_armor/t45d/bos
+	name = "Brotherhood T-45D helmet"
```
Comment (BurpleBineapple) [inline code/modules/clothing/head/f13head.dm:409]:
```suggestion
	name = "Brotherhood T-45d helmet"
```

File: code/modules/clothing/head/f13head.dm [hunk 2]
```diff
@@ -405,6 +405,13 @@
...
+	desc = "(VIII) A T-45D helmet put into use by the Brotherhood of Steel."
```
Comment (BurpleBineapple) [inline code/modules/clothing/head/f13head.dm:410]:
```suggestion
	desc = "(VIII) A T-45d helmet put into use by the Brotherhood of Steel."
```

File: code/modules/jobs/job_types/bos.dm [hunk 3]
```diff
@@ -411,23 +415,20 @@ Star Paladin
...
+		/obj/item/book/granter/trait/big_leagues=1,
 		/obj/item/melee/classic_baton/telescopic=1
 		)
 
 /datum/outfit/loadout/spaladinc
 	name = "Senior Heavy Paladin"
 	backpack_contents = list(
+		/obj/item/gun/energy/laser/aer12=1,
+		/obj/item/stock_parts/cell/ammo/mfc=3,
+		/obj/item/grenade/flashbang=2,
```
Comment (BurpleBineapple) [inline code/modules/jobs/job_types/bos.dm:428]: Not really sure how I feel about these loadouts. The bug with Big Leagues was fixed recently and it does **much** less damage than it used to. There's really no reason you should ever choose a powerfist or sledge over an AER12 or AER14.

Comment (BurpleBineapple) [reply code/modules/jobs/job_types/bos.dm:428]: This also gives the ranged loadout **even more** grenades, which I don't think it is the way to go.

File: code/modules/clothing/head/f13head.dm (hunk 1 above)
Comment (BurpleBineapple) [reply code/modules/clothing/head/f13head.dm:409]: Minor change to match with the case of other armors

File: code/modules/jobs/job_types/bos.dm (hunk 3 above)
Comment (SouthyOsmanli) [reply code/modules/jobs/job_types/bos.dm:428]: I can remove the grenades shortly. I copied ones out. No one uses the grenadesa s is so - easy removal.

**Comment by SouthyOsmanli** [comment]
> Everything looks good, but I do not think Paladins should get big leagues, as it's literally the best PvE perk for a faction that can already steamroll bunkers thanks to their easy access to shields and high tier armor and medicine. This will just allow them to rush bunkers and then use the high tier guns from there to round out their loadout.
> 
> Maybe give them the cut down laser tribeam, for a close-range option in combination with the power fist.

They already can and do with the current unchanged powerfist - and did previously with other weapons. Even before this change BOS star Paladins can rush bunkers and actively do. I tested it and noticed it made the Powerfist basically the same as now but with less AP.

I don't see why it would need to be removed since all of Legion gets the same perk and only 5 roles in Brotherhood /can/ get it. Not actively do unless they select the loadout. Even then it does not help majorly from what I've noticed except in build-styles for Paladins.

File: code/modules/jobs/job_types/bos.dm (hunk 3 above)
Comment (SouthyOsmanli) [reply code/modules/jobs/job_types/bos.dm]: Fiiixed

**Comment by gurking** [comment]
I wasn't talking about the normal tribeam, I mean the weaker cut off version that was made a while ago, it's just missing a sprite.

You also need to remember that perks stay on the character, they can just use a better melee weapon if they find one, or in general for example just be way more effective with glaives, which are already top-tier in PvE.

The legion is /the/ melee focused faction, and from a lot of hours of observing and events they are also really good at PvE thanks to shields and big leagues.

**Comment by SouthyOsmanli** [comment]
There's very few melee weapons that are one-handed to trade the powerfist out for. I don't see a point in it since I can easily make the same argument for other factions as well that can use melee weapons and have melee focused builds.

Brotherhood have melee builds in games as well as heavier builds; and we even get a super sledge starting (which I've removed to avoid better gear) - the only weapons I've noticed better for big-leauges is that.

And even if you get the weaker tribeam, there is really no reason to use the AER-12 over it. The current issue is laser weapons are basically direct upgrades or downgrades from one another, not 'trade-offs'. That's the current issue and the reason why no one takes certain loadouts and aims for meta loadouts.

**Pull request merged**